code/api/src/modules/subscription/types.js

```diff
@@ -15,8 +14,9 @@ const SubscriptionType = new GraphQLObjectType({
     user: { type: UserType },
     crate: { type: CrateType },
     createdAt: { type: GraphQLString },
-    updatedAt: { type: GraphQLString }
+    updatedAt: { type: GraphQLString },
```

Comment:
Storing as a string, maybe for future iterations, we can change to type Date through the graphql-iso-date add-on

Reply:
I have never used the graphql-iso-date but it sounds interesting

Reply:
Nice Work!

Reply:
Good note. Maybe create an issue with a tag for future iteration so there is evidence that this is on the to do list? Ian suggested that last mod for things like this and it was really helpful.